### ISSUE TITLE
request to enable templating

### ISSUE THREAD

**Issue by mathematicalmichael** [binder-examples/jupyter-rise#2]
Github just released a new "use as template" option that something like this would be absolutely perfect for (ditto for all the binder-examples). Any interest in enabling it for this repo?

**Comment by choldgraf** [comment]
I think this is a cool idea! Maybe we should open this up at the BinderHub repository since it would span all of these example repos as well

**Comment by mathematicalmichael** [comment]
Yes that would certainly be helpful. I myself am constantly thinking now about re-use of binder set ups as a way to “capture all the requirements of reproducibility” and frankly just discovered all the examples! I can’t believe they’ve been around this long, too.
